grafana-iml.ini

```diff
@@ -0,0 +1,31 @@
+[database]
+type = postgres
+host = localhost
```

Comment:
possible to connect via socket?

Reply:
I found it: https://github.com/grafana/grafana/issues/2553
I'll update, and pull out all the pgsql changes